test/HeadersTest.php

```diff
@@ -885,11 +872,41 @@ public function testRequestOutWithUseProxyFalse () {
     $_SERVER['REQUEST_URI'] = $env['REQUEST_URI'];
 
     $headers = new Headers($env, $store);
-    $includePath = 'dummy';
-    $headers->requestOut($includePath);
+    $headers->requestOut();
     $this->assertEquals('en.minimaltech.co', $env['HTTP_X_FORWARDED_HOST']);
   }
 
+  public function testRequestOutUrlPatternPath () {
+    $store = $this->createStore();
+    $store->settings['url_pattern_name'] = 'path';
+    $store->settings['url_pattern_reg'] = '\/(?P<lang>[^\/.]+)(\/|\?|$)';
+
+    $env = $this->getEnv('_path');
+    $_SERVER['REQUEST_URI'] = $env['REQUEST_URI'];
+
+    $headers = new Headers($env, $store);
+    $headers->requestOut();
+    $this->assertEquals('/t.php?', $env['REQUEST_URI']);
+    $this->assertEquals('/t.php', $env['REDIRECT_URL']);
```

Comment:
can we check it's here before the requestOut ?

Reply:
Sure. I added.

Reply:
thanks